src/module/vue/components/buttons/btn-rollprogress.vue

```diff
@@ -20,6 +21,16 @@ const props = defineProps<{ item: any; tooltip?: string; disabled?: boolean }>()
 
 const $actor = inject($ActorKey, undefined)
 
+const progressScore = computed(() => {
+  let item: IronswornItem | undefined
+  if ($actor) {
+    item = $actor.items.find((x) => x.id === props.item._id)
+  } else {
+    item = game.items?.get(props.item.id)
+  }
+  return Math.floor(item?.data.data.current / 4)
+})
```

Comment:
This is always used inside a `<progress-box>` component, which [provides](https://github.com/ben/foundry-ironsworn/blob/34158fa789bf3848cd38f8102253bb9f356f1d76/src/module/vue/components/progress/progress-box.vue#L115) a Foundry item for you:

```suggestion
const $actor = inject($ActorKey, undefined)
const $item = inject($ItemKey, undefined)

const progressScore = computed(() => {
  return Math.floor($item?.data.data.current / 4)
})
```

Reply:
oh! yeah, that makes way more sense. yeah, i . given that the same function is coming up twice --

Reply:
oh! right, that makes way more sense. given that the same procedure is showing up multiple times -- what do you think of adding an `Item.progressScore()` method to standardize it? or is that not worth it if it's useless for the other item types?

Reply:
Yeah, that might be nice. It'd have to return `number | undefined` which is unfortunate, but 🤷🏼